### DISCUSSION TITLE
Contributors not updated after successfully merged PR

### DISCUSSION THREAD

**Discussion by nkappler** [community/community#112512] [Pull Requests]
### Select Topic Area

Bug

### Body

A User made the first contribution in nkappler/ctxmenu#51 which was released in [this release](https://github.com/nkappler/ctxmenu/releases/tag/v1.6.2) where the user correctly shows up as new contributor in the automatically generated release notes.

However in the repository overview and the [contributors graph](https://github.com/nkappler/ctxmenu/graphs/contributors), the user doesn't appear at all.

**Comment by abdalla-am** [comment]
It sounds like there's an issue with the user's contributions not being properly recognized in the repository overview and contributors graph despite being acknowledged in the release notes. This could be due to various reasons such as delayed updates, contribution guidelines not being met, or repository settings. I would recommend checking the user's contribution history, refreshing the overview page, ensuring adherence to contribution guidelines, and reviewing repository settings. If the issue persists, reaching out to platform support might be necessary for further assistance.
